time_execution/backends/elasticsearch.py

```diff
@@ -89,9 +99,12 @@ def write(self, name, **data):
         data["name"] = name
         data["timestamp"] = datetime.utcnow()
 
-        self.client.index(
-            index=self.get_index(),
-            doc_type=self.doc_type,
-            id=None,
-            body=data
-        )
+        try:
```

Comment:
If we could use udp for influx that would be great but to my understanding that is not an option and then we will need something of a queue in the middle that can receive udp and sent it over tcp to elastic. Worth investigating, but I agree, in another issue as imporovement :)